reference/shaders-reflection/asm/aliased-entry-point-names.asm.multi.json

```diff
@@ -0,0 +1,49 @@
+{
+    "entryPoints" : [
+        {
+            "name" : "main",
+            "mode" : "vert"
+        },
+        {
+            "name" : "main2",
+            "mode" : "vert"
+        },
+        {
+            "name" : "main",
+            "mode" : "frag"
+        },
+        {
+            "name" : "main2",
+            "mode" : "frag"
+        }
+    ],
+    "types" : {
+        "8" : {
```

Comment:
The member is a string, could it be made an actual integer?

Reply:
The problem is that the members of a struct each have a type field which is either a built-in type, like 'vec4' or a key back into this map.  I wanted to avoid making that field switch between string and integer types.  

I _could_ prefix all these identifiers with underscores to make it more clear they're just opaque IDs.